Source/UnitTests/VideoCommon/VertexLoaderTest.cpp

```diff
@@ -106,6 +106,15 @@ class VertexLoaderTest : public testing::Test
   std::unique_ptr<VertexLoaderBase> m_loader;
 };
 
+template <typename T>
+T clamp(float v)
```

Comment:
Can't `lo` and `hi` be declared as `const` and `clamp` as a `constexpr` function?